portfolio/src/main/webapp/index.html

```diff
@@ -8,10 +8,57 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Deron Gentle's Portfolio</h1>
+
+      <img src="/Images/gradpic.jpg" alt ="graduation picture" width="384" height="512">
```

Comment:
Good job on adding alt text! Good practice :)

Reply:
Thank you